src/components/Pagination.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState, useEffect } from 'react';
+import S from './Pagination.module.css';
+
+export function Pagination({ pageLimit, defaultPage = 1, onChange }) {
+  const [current, setCurrent] = useState(defaultPage);
+
+  useEffect(() => {
+    onChange && onChange(current);
+  }, [current]);
+
+  const handelNextPage = () => {
+    if (current > 1) {
+      setCurrent(next => next + 1);
+    }
+  };
```

Comment:
```suggestion
  const handleNextPage = () => {
    if (current > 1) {
      setCurrent(current => current + 1);
    }
  };
```
오타네요!

그리고 current의 값을 + 1해주는 거니 변수 명도 current로 해주는 것이 맞아보이네요!